### PR TITLE
Bump PlantTrobes GeneFamilyAligner to 1.0.3

### PR DESCRIPTION
**Pull request by gregvonkuster**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/plant_tribes_gene_family_aligner/1.0.3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+source:
+  fn: v1.0.3.tar.gz
+  url: https://github.com/dePamphilis/PlantTribes/archive/v1.0.3.tar.gz
+  md5: fadbcc5f105eb05156e5068380840460
+
+build:
+  number: 0
+
+requirements:
+  build:
```
Comment (bgruening) [inline recipes/plant_tribes_gene_family_aligner/1.0.3/meta.yaml]: Why does this have a build dependency.

File: recipes/plant_tribes_gene_family_aligner/1.0.3/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    - python >=2.7,<3
+  run:
+    - python >=2.7,<3
```
Comment (bgruening) [inline recipes/plant_tribes_gene_family_aligner/1.0.3/meta.yaml]: Can you skip python3 in the build section, please.

**Comment by gregvonkuster** [comment]
Is this how its done @bgruening ?

**Comment by bgruening** [comment]
Yes, but why do you have python as build dependency. In your other recipes you don't have.

**Comment by gregvonkuster** [comment]
@bgruening Each of the PlantTribes packages has different underlying dependencies.  This particular package require pasta which is python based.

**Comment by bgruening** [comment]
@gregvonkuster I'm talking about **build** environments.

**Comment by gregvonkuster** [comment]
@bgruening Thanks! ;)

**Pull request merged**